### ISSUE TITLE
star ground

### ISSUE THREAD

**Issue by hartytp** [sinara-hw/Thermostat#68]
@gkasprow I don't have altium, so it's hard to follow the layout. But, just to check: the thermistor ref gnd should be tied to ADC pin 2 (ref -) but on the schematic it looks like it's tied to C54. Why is this?

**Comment by hartytp** [comment]
@gkasprow can you sketch out the routing of the ref and ref GND to the thermistors for me please?

**Comment by hartytp** [comment]
FWIW I don't understand why IC14 uses the ADC 2V5 reference and REF GND. Why not just use the TEC ref output? Would simplify the routing and ensure that we don't accidentally couple noise form the TEC to the ADC.

Not a big deal and probably not worth changing now, I just don't understand why it was done -- although, I suspect it was a mis-communication between us.

**Comment by hartytp** [comment]
@gkasprow thanks again for all the work on this.

It's hard to follow the layout from the PDFs, but the bits I can follow look really nice. Good work. Will be very curious to see what happens when we stabilise one of our cavities. <100uK RMS?

**Comment by gkasprow** [comment]
![obraz](https://user-images.githubusercontent.com/4325054/69919510-c3670e80-147d-11ea-9972-152bfd01ede5.png)

**Comment by gkasprow** [comment]
I used net ties to make sure the GND is not shorted to GND. Net tie creates a different net names while providing physical connection. ADC ref- goes to GND so it is the same net. I can change on the schematic but it does not change anything on PCB.

**Comment by hartytp** [comment]
Okay that’s pretty good

I would prefer if possible to have the star ground for the ref - right at the ADC ref - pin rather than at the ep. But that’s close

**Comment by gkasprow** [comment]
OK, no problem. It that make you sleep better... :)

**Comment by hartytp** [comment]
Your call. Either route as a proper star ground right to the pin or close now if you think it’s not worth it (in any case the new layout with gp cuts and more layers should make these ground issues less of a provlem)

**Comment by hartytp** [comment]
Thanks! The thing killing my sleep right now is booster

**Comment by gkasprow** [comment]
![obraz](https://user-images.githubusercontent.com/4325054/69919568-92d3a480-147e-11ea-965a-7856b9345179.png)

**Comment by hartytp** [comment]
Perfect

**Comment by gkasprow** [comment]
True, IC should use ADC reference. 2V5 does not much sense here. It is residue from previous revision.

**Comment by gkasprow** [comment]
I mean IC14 and CPU ADC

**Comment by hartytp** [comment]
Indeed. I think that right now the only thing that uses the ADC ref is that opamp and that should probably use the tec ref for convenient (the tec voltage measurement doesn’t need to be particularly high accuracy).

**Comment by hartytp** [comment]
If you can be bothered then fix it as it makes the design neater, but it’s not crutical

**Comment by gkasprow** [comment]
fixed
